### PR TITLE
Warn on console for untrusted destinations

### PR DESCRIPTION
**Pull request by j3parker**
CC @daryl-mcmillan 

A bit speculative; what do you think, Owen?

Addresses #42

### CODE REVIEW

**Comment by omsmith** [comment]
I would make an opts.debug or something. Believe ifrau has similar

**Comment by j3parker** [comment]
Hm, I'm not happy that turning on debugging requires changing the code but I do like conforming to conventions. Sounds right.

File: index.js [hunk 1]
```diff
@@ -49,7 +49,16 @@ module.exports = function(opts) {
...
+			// Relative URLs currently indicate JS running inside  the LE's
+			// origin. We don't need access tokens to call those APIs (the
+			// web session is sufficient.)
+			if (isRelative(parsed)) {
+				finish();
+				return this;
+			}
+
+			if (!isTrusted(parsed, opts.trustedHost)) {
+				console.warn('Not sending access token for ' + parsed + ', opts.trustedHost = ' + opts.trustedHost); // eslint-disable-line no-console
```
Comment (daryl-mcmillan) [inline index.js]: console && console.warn && console.warn() ?

**Comment by j3parker** [comment]
Closing due to inactivity. Probably reasonable and @daryl-mcmillan 's comment looks good. Not sure how much contemporary stuff uses this library vs. the polymer one.

**Pull request closed without merging**